test/DebugInfo/basic.swift

```diff
@@ -3,6 +3,7 @@
 // Verify that we don't emit any debug info by default.
 // RUN: %target-swift-frontend %s -emit-ir -o - \
 // RUN:   | %FileCheck %s --check-prefix NDEBUG
+// NDEBUG: foo
```

Comment:
The FileCheck interpretation of
```
CHECK: a
CHECK-NOT: b
CHECK-NOT: c
```
is "find 'a', then make sure that I don't find 'b' or 'c' after it". It doesn't stop 'b' and 'c' from appearing *before* 'a'.

Reply:
Thanks for clarifying! That's actually what I intended effect here: I just wanted to add a positive check to ensure that we emit LLVM IR at all, in this case the function definition and then make sure no !dbg attachment appears in the function body after that. Since `foo` is a terrible string to search for I changed it to source_filename now, which is guaranteed to appear at the very top of the output.

Reply:
Ah, okay! Sorry, seen too many cases where I add a CHECK line and miss that I just allowed a negative pattern to slip in before it.